### PR TITLE
product listing style update

### PR DESCRIPTION
**Pull request by rohanmathur91**
## Description:
 - Fix minor styling issues

### CODE REVIEW

**Review by anupkgurung — APPROVED** [review]
looks good.

File: style.css [hunk 1]
```diff
@@ -10,18 +10,19 @@
...
+	--gray: #94969f;
+	--light-gray: #e5e5e5;
+	--dark-gray: #666666;
 	--primary-color: #302f34;
 	--primary-hover-color: #4a494e;
 	--secondary-color: #9a9a9a;
 	--secondary-hover-color: #afafaf;
-	--background-color: #fafafa;
+	--background-color: #fff;
 	--navigation-shadow: rgba(0, 0, 0, 0.2);
```
Comment (Naveenchandar) [inline style.css:21]: Please use hex codes for consistency.

**Review by Naveenchandar — APPROVED** [review]
Looks good

File: style.css (hunk 1 above)
Comment (rohanmathur91) [reply style.css:21]: Sure I will update, just curious is there any difference when we write #FFF and #FFFFFF because color is same.

**Pull request merged**